src/daemon/slack/receive.py

```diff
@@ -93,10 +93,20 @@ def start_receive_slack_thread():
             SNS_SLACK_BOT_ACCOUNT,
             is_admin=False)
         slack_user.save()
+    proxies = System.get_request_proxies()
+    if proxies:
```

Comment:
The processing line will be shorter. What do you think?

```
proxy_str = None
if 'https' in proxies:
            proxy_str = proxies['https']
```

Reply:
Thank you. I fixed it.